.travis.yml

```
@@ -297,17 +297,6 @@ jobs:
         - COMPOSER=./tests/composer/composer-php8.json composer install
         - composer show
     - php: nightly
```

```suggestion
```

---

There was no error - I guess it just silently skipped this one?

---

It's easier to understand quantum mechanics than it is to predict how travis will react to any given config.